core/src/trezor/messages.py

```
@@ -414,6 +414,138 @@
+        signature: "bytes"
+
+        def __init__(
+            self,
+            *,
+            public_key: "bytes",
+            signature: "bytes",
+        ) -> None:
+            pass
+
+        @classmethod
+        def is_type_of(cls, msg: Any) -> TypeGuard["BenfenSignedTx"]:
```

Ruff (0.8.2)

Dynamically typed expressions (typing.Any) are disallowed in `msg`

(ANN401)